src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLScalarFunctionsIT.java

```diff
@@ -1,137 +1,126 @@
+                return (final CreateVirtualSchemaTestSetupRequest request) -> {
+                    final Schema postgresSchema = postgresFactory.createSchema(getUniqueIdentifier());
+                    for (final TableRequest tableRequest : request.getTableRequests()) {
+                        final Table.Builder tableBuilder = postgresSchema
+                                .createTableBuilder(tableRequest.getName().toLowerCase());
+                        for (final Column column : tableRequest.getColumns()) {
+                            tableBuilder.column(column.getName().toLowerCase(), column.getType());
+                        }
+                        final Table table = tableBuilder.build();
+                        for (final List<Object> row : tableRequest.getRows()) {
+                            table.insert(row.toArray());
+                        }
+                    }
+
+                    final VirtualSchema virtualSchema = SETUP.createVirtualSchema(postgresSchema.getName(), Map.of());
+
+                    return new PostgreSQLSingleTableVirtualSchemaTestSetup(virtualSchema, postgresSchema);
+                };
             }
-        };
-    }
-
-    @Override
-    protected SingleRowSingleTableVirtualSchemaTestSetup<Timestamp> createDateVirtualSchemaTable() {
-        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
-        return createTestTable("timestamp");
-    }
-
-    @Override
-    protected SingleRowSingleTableVirtualSchemaTestSetup<Integer> createIntegerVirtualSchemaTable() {
-        return createTestTable("integer");
-    }
 
-    @Override
-    protected SingleRowSingleTableVirtualSchemaTestSetup<Double> createDoubleVirtualSchemaTable() {
-        return createTestTable("real");
-    }
+            @Override
+            public String getExternalTypeFor(final DataType exasolType) {
+                switch (exasolType.getExaDataType()) {
+                case VARCHAR:
+                    return "VARCHAR(" + exasolType.getSize() + ")";
+                case DOUBLE:
+                    return "DOUBLE PRECISION";
+                case DECIMAL:
+                    if (exasolType.getScale() == 0) {
+                        return "INTEGER";
+                    } else {
+                        return exasolType.toString();
+                    }
+                default:
+                    return exasolType.toString();
+                }
+            }
 
-    @Override
-    protected SingleRowSingleTableVirtualSchemaTestSetup<Boolean> createBooleanVirtualSchemaTable() {
-        return createTestTable("boolean");
-    }
+            @Override
+            public Set<String> getDialectSpecificExcludes() {
+                return Set.of(
+                        // expected was a value close to <1970-03-01> (tolerance: +/- <0.00010>) but was
+                        // "1970-03-01T00:00:00Z"
+                        "add_months",
```

Comment:
did this happen with the latest version of hamcrest-result-set-matcher? Or did you exclude it before updating?

Reply:
It happened with the latest version.